common/scala/src/main/scala/whisk/http/ErrorResponse.scala

```diff
@@ -248,13 +247,27 @@ object ErrorResponse extends Directives with DefaultJsonProtocol {
   }
 
   implicit val serializer = new RootJsonFormat[ErrorResponse] {
-    def write(er: ErrorResponse) = JsObject("error" -> er.error.toJson, "code" -> er.code.meta.id.toJson)
+    def write(er: ErrorResponse) = {
+      val tidMeta = er.code.meta
+      // Convert the tid to a number. If this is not possible, use `unknown`-tid
```

Comment:
should this serialization of the tid be moved to the transaction id singleton?
also i'd note that the system id should never leak out in the response (there aren't any instances where it will as i remember).